### PR TITLE
Feature - coverage badge using CodeCov

### PR DESCRIPTION
**Pull request by mohituniyal**
The badge in the readme tries to use a coverage file for the master branch. Until this PR is merged, the would not be a cov file for master and therefore the readme on the branch would keep showing `unknown`.

### CODE REVIEW

**Comment by melisabok** [comment]
When I run pytest in my localhost I get this error:

```
(py-3.7) hockney:openreview-py mbok$ pytest
ERROR: usage: pytest [options] [file_or_dir] [file_or_dir] [...]
pytest: error: unrecognized arguments: --cov-report --cov=./
  inifile: /Users/mbok/iesl/openreview-py/pytest.ini
  rootdir: /Users/mbok/iesl/openreview-py
```

what is missing?

I did `pip install -e .` and also fails.

**Comment by melisabok** [comment]
is it necessary to commit .coverage file? 

I have also a lot of other files that start with .coverage, can we add then to the gitignore file?

**Comment by mohituniyal** [comment]
> When I run pytest in my localhost I get this error:
> 
> ```
> (py-3.7) hockney:openreview-py mbok$ pytest
> ERROR: usage: pytest [options] [file_or_dir] [file_or_dir] [...]
> pytest: error: unrecognized arguments: --cov-report --cov=./
>   inifile: /Users/mbok/iesl/openreview-py/pytest.ini
>   rootdir: /Users/mbok/iesl/openreview-py
> ```
> 
> what is missing?
> 
> I did `pip install -e .` and also fails.
> 
> UPDATE: I could make it work installing pytest-cov.

Yes, I have put `pytest-cov` like an optional dependency, similar to the arrangement you designed for pytest-selenium. One would need to separately install it as mentioned in the `Readme.md`

**Comment by mohituniyal** [comment]
> is it necessary to commit .coverage file?
> 
> I have also a lot of other files that start with .coverage, can we add then to the gitignore file?

I have removed the `.coverage` file. It was already added to `gitignore` but somehow still got committed. What other coverage files are you seeing? I ask because just by running pytest, I do not get any other files except for this 1 `.coverage` file

**Comment by melisabok** [comment]
> What other coverage files are you seeing?

They disappeared with the last commit 👍

File: openreview/conference/matching.py [hunk 1]
```diff
@@ -115,7 +115,7 @@ def _create_edge_invitation(self, edge_id):
         edge_head_query = {
             'invitation' : self.conference.get_blind_submission_id()
         }
-        if 'Custom_Load' in edge_id:
```
Comment (melisabok) [inline openreview/conference/matching.py:118]: these commits belong to another PR

**Comment by melisabok** [comment]
> What other coverage files are you seeing? 

They don't appear anymore. problem solved. 👍

File: openreview/conference/matching.py (hunk 1 above)
Comment (mohituniyal) [reply openreview/conference/matching.py:118]: Correct, I am expecting the PR https://github.com/openreview/openreview-py/pull/645 to be merged first. Without these changes the branch would fail the tests.

**Comment by melisabok** [comment]
You have a conflict now.
